packages/node/src/integrations/onuncaughtexception.ts

```diff
@@ -7,6 +7,28 @@ import { logAndExitProcess } from './utils/errorhandling';
 
 type OnFatalErrorHandler = (firstError: Error, secondError?: Error) => void;
 
+interface OnUncaughtExceptionOptions {
+  // TODO(v8): Evaluate whether we should switch the default behaviour here.
+  // Also, we can evaluate using https://nodejs.org/api/process.html#event-uncaughtexceptionmonitor per default, and
+  // falling back to current behaviour when that's not available.
+  /**
+   * Whether the SDK should mimic native behaviour when a global error occurs. Default: `true`
+   * - `false`: The SDK will exit the process on all uncaught errors.
+   * - `true`: The SDK will only exit the process when there are no other 'uncaughtException' handlers attached.
```

Comment:
l: I think we still need to adjust the JSDoc a little. Suggestion but feel free to ignore or do whatever:
 
```suggestion
  /**
   *  Controls if the SDK should mimic Node-native behaviour when a global error occurs:
   * - `false`: The SDK will exit the process on all uncaught errors.
   * - `true`: The SDK will only exit the process when there are no other `uncaughtException` handlers attached.
  
    Default: `true`
```

Reply:
Added. Thank you!